at_client/src/main/java/org/atsign/client/cli/Register.java

```diff
@@ -24,16 +24,16 @@ public static void main(String[] args) throws Exception {
         String otp;
         String validationResponse;
         String cramSecret;
-        String rootDomain = configReader.getProperty("ROOT_DOMAIN");
-        String rootPort = configReader.getProperty("ROOT_PORT");
-        String registrarUrl = configReader.getProperty("REGISTRAR_URL");
-        String apiKey = configReader.getProperty("API_KEY");
-        
-        if (rootDomain == null || rootPort == null || registrarUrl == null || apiKey == null){
+        String rootDomain = configReader.getProperty("rootServer", "domain");
```

Comment:
Add here, for backwards compatibility 
```
if (rootDomain == null) {
  rootDomain = configReader.getProperty("ROOT_DOMAIN");
}
```
and similar for each of the next three config properties